core/renderers/zelos/drawer.js

```diff
@@ -143,8 +143,26 @@ Blockly.zelos.Drawer.prototype.drawFlatBottom_ = function() {
  * @override
  */
 Blockly.zelos.Drawer.prototype.drawInlineInput_ = function(input) {
-  // Don't draw an inline input.
   this.positionInlineInputConnection_(input);
+
+  var inputName = input.input.name;
+  if (input.connectedBlock || this.info_.isInsertionMarker) {
+    return;
+  }
+
+  var width = input.width - input.connectionWidth;
+  var height = input.height;
+  var yPos = input.centerline - height / 2;
+
+  var connectionRight = input.xPos + input.connectionWidth;
+
+  var outlinePath = Blockly.utils.svgPaths.moveTo(connectionRight, yPos) +
+      Blockly.utils.svgPaths.lineOnAxis('h', width - input.connectionWidth) +
```

Comment:
You're defining width up above:
`var width = input.width - input.connectionWidth`
then only using it with an extra subtraction:
`width - input.connectionWidth`

So line 153 can become
`var width - input.width - 2 * input.connectionWidth`
 and the later references can just use `width`.

Reply:
Good catch.